# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xwayland/no-shared-xorg.patch
# Copyright (C) 2022 - 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xwayland-22.1.3/meson.build.vanilla	2022-07-12 16:28:16.067678760 +0200
+++ xwayland-22.1.3/meson.build	2022-07-12 16:33:14.727694167 +0200
@@ -568,9 +568,3 @@
 if host_machine.system() != 'windows'
     subdir('test')
 endif
-
-install_man(configure_file(
-    input: 'man/Xserver.man',
-    output: 'Xserver.1',
-    configuration: manpage_config,
-))
